teamB/kimSungHoon/routes/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+
+const userRouter = require('./userRouter');
+router.use('/users', userRouter.router);
+
+const postRouter = require('./postRouter');
+router.use('/posts', postRouter.router);
```

Comment:
가독성을 위해 모듈을 불러오는 부분과 모듈 동작에 필요한 프로퍼티를 추가하는 부분을 분리해주세요!
```suggestion
const userRouter = require('./userRouter');
const postRouter = require('./postRouter');

router.use('/users', userRouter.router);
router.use('/posts', postRouter.router);
```

Reply:
수정완료 했습니다.